feature/sessions/src/commonMain/kotlin/io/github/droidkaigi/confsched/sessions/components/TimetableItemDetailFloatingMenu.kt

```diff
@@ -71,16 +78,24 @@ fun TimetableItemDetailFloatingActionButtonMenu(
 private fun TimetableItemDetailFloatingActionButtonMenu(
     expanded: Boolean,
     isBookmarked: Boolean,
+    fabBookmarkFlag: Boolean,
     slideUrl: String?,
     videoUrl: String?,
     onExpandedChange: (Boolean) -> Unit,
     onBookmarkClick: (isBookmarked: Boolean) -> Unit,
+    onFabBookmarkFlagChange: (Boolean) -> Unit,
     onAddCalendarClick: () -> Unit,
     onShareClick: () -> Unit,
     onViewSlideClick: (url: String) -> Unit,
     onWatchVideoClick: (url: String) -> Unit,
     modifier: Modifier = Modifier,
 ) {
+    var height by remember { mutableStateOf(0.dp) }
+    val density = LocalDensity.current
+    LaunchedEffect(height) {
+        onFabBookmarkFlagChange(isBookmarked)
+    }
```

Comment:
[nitpick] We might want to add a small delay here to debounce size changes.  
Also, adding a short comment to explain the purpose of this effect could help other contributors understand the background of this code.

For example:
```
// Recompose child menu items only after the view size has settled.
// Recomposing them during the transition may cause a brief flicker that makes the menu look flaky.
```